app/views/layouts/login.html.haml

```diff
@@ -1,4 +1,4 @@
-!!!
+= render :partial => 'layouts/doctype'
```

Comment:
I find this really weird to have a partial just for the doctype, but I guess it does keep the official one in one place so you can change it all over at once.